<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 83, 119);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>1181</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 253, 255);
border-color: rgb(235, 253, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="SearchBook" name="searchBook">
    <attribute name="title">
     <string>Search For A Book</string>
    </attribute>
   </widget>
   <widget class="Login" name="login">
    <attribute name="title">
     <string>Log In</string>
    </attribute>
   </widget>
   <widget class="BorrowedBooksList" name="borrowedBookList">
    <attribute name="title">
     <string>MyAccount</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SearchBook</class>
   <extends>QWidget</extends>
   <header>SearchBook.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Login</class>
   <extends>QWidget</extends>
   <header>Login.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BorrowedBooksList</class>
   <extends>QWidget</extends>
   <header>borrowedbookslist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
